src/pydata_sphinx_theme/__init__.py

```diff
@@ -140,18 +141,19 @@ def update_config(app):
     # Add extra icon links entries if there were shortcuts present
     # TODO: Deprecate this at some point in the future?
     icon_links = theme_options.get("icon_links", [])
-    for url, icon, name in shortcuts:
-        if theme_options.get(url):
-            # This defaults to an empty list so we can always insert
-            icon_links.insert(
-                0,
-                {
-                    "url": theme_options.get(url),
-                    "icon": icon,
-                    "name": name,
-                    "type": "fontawesome",
-                },
-            )
+    if icon_links is not None:
+        for url, icon, name in shortcuts:
```

Comment:
same question as above: the code on `main` is fine if `icon_links` is not defined, it only errors if you explicitly set `icon_links=None` in the `html_theme_options` dictionary of `conf.py`.  So why do that?